avidemux_plugins/ADM_scriptEngines/tinyPy/src/binding/adm.admPyClass

```diff
@@ -31,7 +31,7 @@
 /* METHOD */ int  pySetFilm2Pal:audioSetFilm2Pal(int,int)
 /* METHOD */ int  pySetNormalize:audioSetNormalize(int,int,int)
 /* METHOD */ int  pySetNormalize2:audioSetNormalize2(int,int,int,int)
-/* METHOD */ int  pySetDrc:audioSetDrc(int,int)
+/* METHOD */ int  pySetDrc:audioSetDrc(int,int,int,float,float,float,float,float)
```

Comment:
Rather than startle users who try to run old project scripts with a tinyPy exception, maybe it is worth preserving audioSetDrc signature and turning the function into a wrapper for audioSetDrc2 (or audioSetDrcEx or whatever != old name) with all the additional args?

Reply:
[66735c5](https://github.com/mean00/avidemux2/pull/417/commits/66735c5403bcbdbaf4807da47a5cce4af3004288)